lnwallet/wallet.go

```diff
@@ -432,13 +439,21 @@ func (l *LightningWallet) InitChannelReservation(
 // validate a funding reservation request.
 func (l *LightningWallet) handleFundingReserveRequest(req *InitFundingReserveMsg) {
 	// It isn't possible to create a channel with zero funds committed.
-	if req.LocalFundingAmt+req.RemoteFundingAmt == 0 {
+	if req.LocalFundingAmt+req.RemoteFundingAmt+req.LocalSpendAmt == 0 {
 		err := ErrZeroCapacity()
 		req.err <- err
 		req.resp <- nil
 		return
 	}
 
+	if req.LocalFundingAmt+req.RemoteFundingAmt > 0 && req.LocalSpendAmt > 0 {
+		err := fmt.Errorf("Local and remote fund amt must be zero " +
```

Comment:
Nit: errors returned should begin lowercase.

There are some other cases as well.

Reply:
Fixed.